README.md

```diff
@@ -0,0 +1,69 @@
+## Manjaro HardWare Detection (mhwd)
```

Comment:
I think it's fine for now. It's a good idea to finally have a proper readme :+1: